graphql-server/pkg/queries/reaction.go

```diff
@@ -60,6 +69,36 @@ func (q *ReactionQuery) ScopeProposals(proposalIds []int) IReactionQuery {
 	return &newQuery
 }
 
+func (q *UserReactionQuery) QueryUserReactions() ([]string, error) {
+	type ReactionMap struct {
+		TargetID string `json:"target_id"`
+		Reaction string `json:"reaction"`
+	}
+	query := q.NewQuery().ColumnExpr("target_id, reaction")
+
+	reactionLists := make([]ReactionMap, 0, len(q.scopedTargetIds))
```

Comment:
I imagine that would be done by another query

orginally I wanted to have something like but i reckon it would be too complex
```
type IUserReactionQuery interface {
        QueryUserReactions() (*models.Reaction, error)
}
type ITargetUserReactionQuery interface {
        QueryUserTargetReactions() (string[], error)
}
type ITargetQuery interface {
	ScopeUserAddress(address string) ITargetUserReactionQuery
	QueryTargetReactions() ([]map[string]int, error)
}
type IReactionQuery interface {
	ScopeTarget(...) ITargetQuery
	ScopeUserAddress(address string) IUserReactionQuery
}
```

Reply:
maybe I will make the return type `*models.Reaction` and convert to string at resolver level